docs/configuration/ai-analysis.rst

```diff
@@ -299,36 +299,320 @@ Advanced - Customizing HolmesGPT
+
+.. code-block:: bash
+
+    helm upgrade robusta robusta/robusta --values=generated_values.yaml --set clusterName=<YOUR_CLUSTER_NAME>
+
 
+You can override fields from the default toolsets to customize them for your needs.
+For example:
+
+.. code-block:: yaml
     enableHolmesGPT: true
     holmes:
-      additionalEnvVars:
-        - name: ROBUSTA_AI
-          value: "true"
       toolsets:
-        # Name of the toolset (for example "mycompany/internal-tools")
-        # Used for informational purposes only (e.g. to print the name of the toolset if it can't be loaded)
-        - name: "resource_explanation"
-          # List of tools the LLM can use - this is the important part
+      confluence:
+      description: "Enhanced Confluence toolset for fetching and searching pages."
+      prerequisites:
+        - command: "curl --version"
+        - env:
+          - CONFLUENCE_USER
+          - CONFLUENCE_API_KEY
+          - CONFLUENCE_BASE_URL
+      tools:
+      - name: "search_confluence_pages"
+        description: "Search for pages in Confluence using a query string."
+        user_description: "search confluence for pages containing {{ query_string }}"
+        command: "curl -u ${CONFLUENCE_USER}:${CONFLUENCE_API_KEY} -X GET -H 'Content-Type: application/json' ${CONFLUENCE_BASE_URL}/wiki/rest/api/content/search?cql=text~{{ query_string }}"
+
+      - name: "fetch_pages_with_label"
+        description: "Fetch all pages in Confluence with a specific label."
+        user_description: "fetch all confluence pages with label {{ label }}"
+        command: "curl -u ${CONFLUENCE_USER}:${CONFLUENCE_API_KEY} -X GET -H 'Content-Type: application/json' ${CONFLUENCE_BASE_URL}/wiki/rest/api/content/?expand=body.storage&label={{ label }}"
+
+
+
+How to define a new toolset?
+-------------------------------------
+A toolset is defined in your Helm values (``generated_values.yaml``). Each toolset has a unique name and has to contain tools.
+.. code-block:: yaml
+
+    toolsets:
+      <toolset_name>:
+        enabled: <true|false>
+        name: "<string>"
+        description: "<string>"
+        docs_url: "<string>"
+        icon_url: "<string>"
+        tags:
+          - <cli|cluster|core>
+        installation_instructions: "<string>"
+        variables:
+          <variable_name>: "<value>"
+        prerequisites:
+          - command: "<shell_command>"
+            expected_output: "<expected output of the command>"
+          - env:
+            - "<environment variable>"
+        additional_instructions: "<string>"
+        tools:
+          - name: "<string>"
+            description: "<string>"
+            command: "<shell command template>"
+            script: "<script content>"
+            parameters:
+              <parameter_name>:
+                type: "<string>"
+                description: "<string>"
+                required: <true|false>
+
+Toolset Fields
+--------------
+
+.. list-table::
```

Comment:
this doesn't render right in the preview url - see https://docs.robusta.dev/holmes-docs-enable-custom-tools/configuration/ai-analysis.html#how-to-define-a-new-toolset